docs/development/environment/docker.md

```diff
@@ -26,3 +28,8 @@ The `3.x-dev` image also provides the following:
    container. If your user ID is not `1000`, build the image with:
    `--build-arg WWW_DATA_ID=$(id -u)`
     - Default: `1000`
+
+To build the development image, you will have to define the target farmos-dev,
+for example:
+
+`docker build --build-arg WWW_DATA_ID=$(id -u)  -t farmos/farmos:3.x-dev --target farmos-dev docker`
```

Comment:
`--target farmos-dev`
maybe cleaner would be `--target dev`, generally in the Dockerfile I think that prefixing stages with `farmos` is a bit redundant since the entire Dockerfile is for farmOS